src/libs/actions/Report.js

```diff
@@ -706,14 +706,22 @@ function deleteReportComment(reportID, reportAction) {
         },
     };
 
-    // If we are deleting the last visible message, let's find the previous visible one and update the lastMessageText in the LHN.
+    // If we are deleting the last visible message, let's find the previous visible one (or set an empty one if there are none) and update the lastMessageText in the LHN.
     // Similarly, if we are deleting the last read comment we will want to update the lastVisibleActionCreated to use the previous visible message.
-    const lastMessageText = ReportActionsUtils.getLastVisibleMessageText(reportID, optimisticReportActions);
-    const lastVisibleActionCreated = ReportActionsUtils.getLastVisibleAction(reportID, optimisticReportActions).created;
-    const optimisticReport = {
-        lastMessageText,
-        lastVisibleActionCreated,
+    let optimisticReport = {
+        lastMessageText: '',
+        lastMessageHtml: '',
+        lastActorEmail: '',
+        lastVisibleActionCreated: '',
```

Comment:
```suggestion
        lastMessageText: '',
        lastVisibleActionCreated: '',
```
I might be misunderstanding the desired behavior here, but I think we want it to stay with `No activity yet` or the name of the workspace, rather than switching over to the name of the account. It shouldn't show the last actor, since there isn't one. I believe removing these two lines does the trick.

Reply:
@Gonals thoughts?

Reply:
The issue seems to indicate the desired behavior is the other way around (do not show `No activity yet`, as there was some activity), but I can see a case for the opposite. I am ok with either approach, tbh

Reply:
That's a good point! I think I'd still lean towards showing `No activity yet` because I think it might be confusing for someone if they deleted their message to still see their name. For the user, it's as if nothing has happened.

Reply:
Heh. There's actually **another** [issue](https://github.com/Expensify/App/issues/16708) tackling the issue from that point of view, so I just grabbed and will fix both here. I think you are right and that approach makes more sense.